src/modules/core/filter_box_blur.yml

```diff
@@ -0,0 +1,35 @@
+schema_version: 7.0
+type: filter
+identifier: box_blur
+title: Box Blur
+version: 3
+copyright: Meltytech, LLC
+license: LGPLv2.1
+language: en
+tags:
+  - Video
+parameters:
+
+  - identifier: hradius
+    title: Horizontal radius
+    description: >
+      The horizontal blur radius as a percent of the image size.
+      100% results in a blur radius of 10% of the image width.
+    type: integer
```

Comment:
I am confused a little about integer and percent here because it is being get-ed in the code as a double, and some projects such as frei0r use [0.0, 1.0] as a range for a percentage. I understand it a little better from the description but still kind of awkward. Since there is no maximum, then I can use 1000% for 100% of the image dimension. So, why not keep it simpler and use [0.0, 1.0] or [0.0, 100.0] where 100% = 100% and let me use higher precision than 0.1% of the image dimension? I think you are trying to map the most usable portion of the range into [0, 100] but also not make it limited. I think that makes sense for a UI, and this can simplify a UI for this; but not so sure it makes sense here. I will not object but at the very least, change the type to float.

Reply:
>  I think you are trying to map the most usable portion of the range into [0, 100] but also not make it limited. I think that makes sense for a UI, and this can simplify a UI for this

You understand my dilemma perfectly. If I set the maximum range to be 100% of the image width, then in the UI I need to scale that down by 1/10th because 100% image blur is not useful in the application (and all the useful values would be in the first 10% of the slider). 

I was hoping you would offer a stronger opinion and then I would change it to whatever your preference is. I agree with changing the type to float. But let me know if you think of a better range/scale for this parameter.

Reply:
Changed the type to float.